letsencrypt/config.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Let's Encrypt",
-  "version": "4.2",
+  "version": "4.4",
```

Comment:
4.3?

Reply:
Yes, changelog included 4.3 already. seems this was missed to be updated in this file before